.github/CODEOWNERS

```diff
@@ -140,6 +140,9 @@
 # ServiceLabel: %AVS %Service Attention
 /sdk/avs/Microsoft.Azure.Management.Avs/                           @cataggar @nivasn-msft @amitchat @aishu
 
+# ServiceLabel: %Azure Carbon Optimization
```

Comment:
As this is a management package, you'll want to put this down in the management section around [L930](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS#L930) and follow the formatting there.   This should look something like:

```text
# PRLabel: %Carbon Optimization
/sdk/carbon/Azure.ResourceManager.CarbonOptimization                  @destinyboyer @liangchenmicrosoft

# ServiceLabel: %Carbon Optimization %Mgmt
# ServiceOwners:                                                      @destinyboyer
```

I'll create the service label as "Carbon Optimization"

Reply:
Done.